lumicks/pylake/correlated_stack.py

```diff
@@ -115,6 +115,25 @@ def crop_by_pixels(self, x_min, x_max, y_min, y_max):
         data = self.src.with_roi(np.array([x_min, x_max, y_min, y_max]))
         return self.from_dataset(data, self.name, self.start_idx, self.stop_idx)
 
+    def define_tether(self, point_1, point_2):
+        """Returns a copy of the stack rotated such that the tether defined by `point_1` and
+        `point_2` is horizontal.
+
+        Parameters
+        ----------
+        point_1 : (float, float)
+            (x, y) coordinates of the tether start point
+        point_2 : (float, float)
+            (x, y) coordinates of the tether end point
+        """
+        if self.src._tether:
+            raise AttributeError(
+                "A tether is already defined; re-defining the coordinates is not allowed."
```

Comment:
It would be good to mention why it's not allowed.

Reply:
well ... I managed to work out how to properly handle any permutation of cropping/tether definition so this will no longer throw. I've requested re-review just for the last commit because enough changed since your initial approval to warrant a quick look through (all changes in the final commit)